content/react/sprints/4/day-plan/index.md

```diff
@@ -7,7 +7,6 @@ weight = 3
 backlog= 'Module-React'
 backlog_filter= 'Week 4'
 [[blocks]]
-name="Progress check-in"
```

Comment:
Why are we removing the Tech Ed content on this?

Reply:
I didn't change anything tech-related, not sure how to even do that. If I did it was not intentional, sorry.

Reply:
Hey! We noticed this PR on the Tech Ed board so I thought I'd take a look. I'm not quite sure exactly what happened here - but Karen's right that this was removing a Tech Ed exercise.

I've now fixed it. You can tell because the change is now only in green - showing that you're just adding a new block, not replacing an existing block.

Reply:
Checking [the preview](https://deploy-preview-370--cyf-curriculum.netlify.app/react/sprints/4/day-plan/) this now looks good to go :)

Reply:
Hey @40thieves thanks, not sure what happened :)